control/tests/nlsys_test.py

```diff
@@ -19,7 +19,7 @@
 # Basic test of nlsys()
 def test_nlsys_basic():
     def kincar_update(t, x, u, params):
-        l = params.get('l', 1)  # wheelbase
+        l = params['l']             # wheelbase
```

Comment:
```suggestion
        l = params['l']              # wheelbase
```
1 space more to align with "x velocity" on line 24.